SystemSettings.php

```diff
@@ -109,7 +122,7 @@ private function createRedisHostSetting()
             $field->inlineHelp = Piwik::translate('QueuedTracking_RedisHostFieldHelp') . '</br></br>'
                 . Piwik::translate('QueuedTracking_RedisHostFieldHelpExtended') . '</br>';
 
-            if ($self->isUsingSentinelBackend()) {
+            if ($self->isUsingSentinelBackend() or $self->isUsingClusterBackend()) {
```

Comment:
Is there a specific reason to use `or` instead of `||`? We typically use `||` unless there's a specific reason to do otherwise.

Reply:
i'll fix it. there is no specific reason, this is nothing but confusion in my dialect, because sometimes i have to jump from PHP to Python back and forth 😄

Reply:
this is also fixed, can you confirm?

Reply:
Looks good. Thank you :+1: